tests/unit/test_logging.py

```diff
@@ -20,7 +20,6 @@ def _(command, expect_error=False, log_level=None, supported_interface=True, **k
         kwargs["env"] = {
             **kwargs.get("env", {k: v for k, v in os.environ.items() if k not in UNSET_VARS}),
             "__BIRDHOUSE_SUPPORTED_INTERFACE": str(supported_interface),
-            "TERM": os.getenv("TERM", "linux"),
```

Comment:
change this line please, don't delete it: https://github.com/bird-house/birdhouse-deploy/pull/555#discussion_r2162410618

Reply:
Can you be more explicit about how you want it changed?